jupyterhub/tests/test_pages.py

```diff
@@ -207,6 +207,16 @@
         assert FormSpawner.options_form in r.text
 
 
+async def test_spawn_page_blank_callable(app):
+    with mock.patch.dict(app.users.settings, {'spawner_class': BlankCallableFormSpawner}):
```

Comment:
missing import of the class

Reply:
Yepp I started debugging this locally, btw the CONTRIBUTION docs are great and got me running without any slow steps.

I ended up running into a redirection loop, below is a processed output of the logs...

```
hub/spawn
user/erik/
hub/user/erik/
hub/spawn?next=hub/user/erik/
user/erik/?next=hub/user/erik/
hub/user/erik/?next=hub/user/erik/
hub/spawn?next=hub/user/erik/%3Fnext%3D%252F%2540%252Fspace%252520word%252Fhub%252Fuser%252Ferik%252F
user/erik/?next=hub/user/erik/%3Fnext%3D%252F%2540%252Fspace%252520word%252Fhub%252Fuser%252Ferik%252F
hub/user/erik/?next=hub/user/erik/%3Fnext%3D%252F%2540%252Fspace%252520word%252Fhub%252Fuser%252Ferik%252F
hub/spawn?next=hub/user/erik/%3Fnext%3D%252F%2540%252Fspace%252520word%252Fhub%252Fuser%252Ferik%252F%253Fnext%253D%25252F%252540%25252Fspace%25252520word%25252Fhub%25252Fuser%25252Ferik%25252F
```

And here is the first few lines of the raw logs showing how the redirection loops starts out.

```
[D 33:29.618 MockHub user:217] Creating <class 'jupyterhub.tests.mocking.BlankCallableFormSpawner'> for erik:
[I 33:29.630 MockHub log:158] 302 GET /@/space%20word/hub/spawn -> /@/space%20word/user/erik/ (erik@127.0.0.1) 21.04ms
[I 33:29.635 MockHub log:158] 302 GET /@/space%20word/user/erik/ -> /@/space%20word/hub/user/erik/ (@127.0.0.1) 0.90ms
[I 33:29.651 MockHub log:158] 302 GET /@/space%20word/hub/user/erik/ -> /@/space%20word/hub/spawn?next=%2F%40%2Fspace%2520word%2Fhub%2Fuser%2Ferik%2F (erik@127.0.0.1) 10.38ms
[I 33:29.672 MockHub log:158] 302 GET /@/space%20word/hub/spawn?next=%2F%40%2Fspace%2520word%2Fhub%2Fuser%2Ferik%2F -> /@/space%20word/user/erik/?next=%2F%40%2Fspace%2520word%2Fhub%2Fuser%2Ferik%2F (erik@127.0.0.1) 13.40ms
```